index.html

```diff
@@ -1,12 +1,129 @@
+          <a class="navigation-link" href="#">О Седоне</a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="catalog.html">Гостиницы</a>
+        </li>
+      </ul>
+      <ul class="navigation-user">
+        <li class="navigaton-item">
+          <a class="navigation-link" href="#">
+            <span class="visually-hidden">Поиск.</span>
+          </a>
+        </li>
+        <li class="navigaton-item">
+          <a class="navigation-link" href="#">
+            <span class="visually-hidden">Избранное.</span>
+          </a>
+        </li>
+        <li class="navigation-item">
+          <a class="navigation-link" href="#">
+            <span class="visually-hidden">Хочу сюда.</span>
+          </a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+
+  <main class="main-invitation">
+
+    <!--Вставлю картинку в 3-м дз -->
+    <section class="preview">
+      <h1>Седона — небольшой городок в Аризоне,<br>заслуживающий большего!</h1>
+      <p>Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
+      <ol class="bloсks">
+        <li class="block1">
+          <h2>Настоящий городок</h2>
+          <p>Седона — не аттракцион<br> для туристов, там течёт<br>своя жизнь</p>
+          <img src="" alt="Фасад домов.">
+          <ul class="mini-block">
+            <li class="mini1">
+              <img src="" alt="Дом.">
+              <h3>Жильё</h3>
+              <p>Рекомендуем пожить<br>в настоящем мотеле,<br>всё как в кино!</p>
+            </li>
+            <li class="mini2">
+              <img src="" alt="Бургер.">
+              <h3>Еда</h3>
+              <p>Всегда заказывайте<br>топовый фирменный бургер,<br> вы не разочаруетесь!</p>
+            </li>
+            <li class="mini3">
+              <img src="" alt="Перевязанная коробка.">
+              <h3>Сувениры</h3>
+              <p>Не только китайского,<br>но и настоящего местного<br>производства!</p>
+            </li>
+          </ul>
+        </li>
+        <li class="block2">
+          <img src="" alt="Скалы каньона.">
+          <h2>Там есть<br>мост дьявола</h2>
+          <p>Да, по нему можно пройти!<br> Если вы осмелитесь,<br> разумеется</p>
+        </li>
+        <li class="block3">
+          <h2>Небольшая<br>площадь</h2>
+          <p>Все достопримечательности<br>находятся очень близко</p>
+        </li>
+        <li class="block4">
+          <h2>Красивая<br>дорога</h2>
+          <p>Ехать в Седону из Лас-Вегаса<br>совсем не скучно!</p>
+        </li>
+        <li class="block5">
+          <h2>Мало туристов</h2>
+          <p>Большинство едет в Гранд Каньон<br>и толпится там</p>
+        </li>
+      </ol>
+    </section>
+    <section class="demand">
+      <h2>Заинтересовались?</h2>
+      <p>Укажите предполагаемые даты поездки,<br>и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+      <button class="button order-button" type="submit">ПОИСК ГОСТИНИЦЫ В СЕДОНЕ</button>
```

Comment:
```suggestion
      <button class="button order-button" type="button">ПОИСК ГОСТИНИЦЫ В СЕДОНЕ</button>
```
когда кнопка ничего не отправляет, а отправить - по-английски- submit, то её тип становится button - те, это просто кнопка, ну как в сериале: просто Мария.